check_license/file-exceptions.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh -e
+# Copyright (C) 2024 Intel Corporation
+# Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# You can add an exception file
+grep -v -E -e '/src/uthash/.*' \
```

Comment:
pls remove the first `/` - we should rather use relative paths and this one my be interpreted as absolute path

Reply:
done